cypress/integration/admin/pageBuilder/blocks/pageBlocksExportImport.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Blocks Export/Import", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllBlocks();
+        cy.pbDeleteAllBlockCategories();
+    });
+
+    const blockNames1 = ["Block1Name"];
+
+    const blockNames2 = ["Block1Name", "Block2Name"];
+
+    const blockNames3 = ["Block1Name", "Block2Name", "Block3Name"];
+
+    //Data used for creating multible block categories
+    const blockCategoryData1 = {
+        name: nanoid(10).toLowerCase(),
+        slug: nanoid(10).toLowerCase(),
+        icon: "icon-name",
+        description: nanoid(10).toLowerCase()
+    };
+
+    const blockCategoryData2 = {
+        name: nanoid(10).toLowerCase(),
+        slug: nanoid(10).toLowerCase(),
+        icon: "icon-name",
+        description: nanoid(10).toLowerCase()
+    };
+
+    const blockCategoryData3 = {
+        name: nanoid(10).toLowerCase(),
+        slug: nanoid(10).toLowerCase(),
+        icon: "icon-name",
+        description: nanoid(10).toLowerCase()
+    };
+
+    it("Test the importation and exportation of all blocks", () => {
+        cy.visit("/page-builder/page-blocks");
+        // Exports all created data and saves the exported string value.
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData1,
+            blockNames: blockNames1
+        });
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData2,
+            blockNames: blockNames2
+        });
+        cy.pbCreateCategoryAndBlocks({
+            blockCategory: blockCategoryData3,
+            blockNames: blockNames3
+        });
+
+        cy.findByPlaceholderText("Search blocks").should("exist");
+        cy.findByTestId("pb-blocks-list-options-menu").click();
+        cy.findByText("Export all blocks").click();
+        cy.findByText("Your export is now ready!").should("exist");
+
+        cy.get("span.link-text.mdc-typography--body2")
```

Comment:
Do we have a test ID available here? 

Please let's not use CSS class selectors.

Reply:
We do have an available test ID but it's available only at the top level of the element hierarchy. In other words we could use a test-id directly on the textbox as right now with the test-id we return more data than we would like. Example for: export-blocks.export-ready-dialog
This get returned: **Your export is now ready!Copy the export URL:https://wby-fm-bucket-ffc8435.s3.amazonaws.com/EXPORTS/8lnaey4t1-WEBINY_BLOCK_EXPORT.zip?AWSAccessKeyId=ASIAZ2AFH5523X4KMNSD&Expires=1696948009&Signature=2MWlzTTGwqa2v%2BDKTRF%2BAZ6EmAA%3D&X-Amzn-Trace-Id=Root%3D1-651c249e-1df988087528cd4b2c10d3a4%3BParent%3D13af8f904a1a8ae9%3BSampled%3D0%3BLineage%3De50262ad%3A0%7C50628ad9%3A6%7C2d7024d9%3A0&x-amz-security-token=IQoJb3JpZ2luX2VjECcaCXVzLWVhc3QtMSJIMEYCIQDAfl6j5%2Fh%2Fr1j0RtdHDHCIoCslXfQNHdftin9XrBX2VAIhAOGAu3HbYy5lM4FlCZDE9oOMOosIkTPG20bZUTGDxfsDKo8DCC8QAxoMNjc0MzIwODcxMjg1IgxMK81UKWwSJfxtMeYq7AIklmNOVdxQb2onultjrpptas9Ljblry09Laz%2FhaudJKHdlYANXng9CCdOrgmBHhdDtpaKyvj%2F35OBVuEoLAox0qlxnicMzEXfPbGBEVbRK4%2Fw6iIiHC%2F9Q%2BovpAeoGAb4sPILJEMLaMhcRQaJTr48iimnOkkJXV98pMLEdAObBXlGp0mWqavqXIEsNKCVq6TMgb%2BIeLDtvuZDIWSGsOJy97Vp%2FJTE6M4MI8ZmfTGjT9Ys5wHB46AOE%2BBlQmoTT2wyXV17DSzInZCeT1V1duubmEvau5XDEAlvFHzjmueXClCSeY%2BcXyuDnxpGGUYAGSwCiPZfk8mSnuBudAANqx%2FQWpRN7NIZWml9OtVzbS8eQ5V14Xla6ykr%2BZEhyzqOYsH%2F4L7S178ty0FY0KW6hBUyQgCSbSDngWSI%2FdaqJ%2FXadIbKlUcjR7JPDM%2Bds%2F8Gw9c79ckW8k72NZ7Zm%2B%2BNeONIhV%2FsXOtBms4cDuZ0XMNq%2F8KgGOpwB6SMup4p3FPGT1Oeanbig542TIVVZVoIDHl5Z72F%2FebMQupiA55NV6hzaivhXzNUnbGq0Ci9ghwDqCjnPSVUmNAWRVf1EYpDFd5zHUDfhCw43J%2BIZeUkInUBAxVA4RmvEqsoVUvA%2FZ21BQlYc3oBqFZ2%2F9oXI9JETONCncmJtUxF0n82bvXjKRdPtbf45PxJj4NK3THZGDSjf6DLNOr download the ZIP archive:DownloadClose**Your export is now ready!Copy the export URL:https://wby-fm-bucket-ffc8435.s3.amazonaws.com/EXPORTS/8lnaey4t1-WEBINY_BLOCK_EXPORT.zip?AWSAccessKeyId=ASIAZ2AFH5523X4KMNSD&Expires=1696948009&Signature=2MWlzTTGwqa2v%2BDKTRF%2BAZ6EmAA%3D&X-Amzn-Trace-Id=Root%3D1-651c249e-1df988087528cd4b2c10d3a4%3BParent%3D13af8f904a1a8ae9%3BSampled%3D0%3BLineage%3De50262ad%3A0%7C50628ad9%3A6%7C2d7024d9%3A0&x-amz-security-token=IQoJb3JpZ2luX2VjECcaCXVzLWVhc3QtMSJIMEYCIQDAfl6j5%2Fh%2Fr1j0RtdHDHCIoCslXfQNHdftin9XrBX2VAIhAOGAu3HbYy5lM4FlCZDE9oOMOosIkTPG20bZUTGDxfsDKo8DCC8QAxoMNjc0MzIwODcxMjg1IgxMK81UKWwSJfxtMeYq7AIklmNOVdxQb2onultjrpptas9Ljblry09Laz%2FhaudJKHdlYANXng9CCdOrgmBHhdDtpaKyvj%2F35OBVuEoLAox0qlxnicMzEXfPbGBEVbRK4%2Fw6iIiHC%2F9Q%2BovpAeoGAb4sPILJEMLaMhcRQaJTr48iimnOkkJXV98pMLEdAObBXlGp0mWqavqXIEsNKCVq6TMgb%2BIeLDtvuZDIWSGsOJy97Vp%2FJTE6M4MI8ZmfTGjT9Ys5wHB46AOE%2BBlQmoTT2wyXV17DSzInZCeT1V1duubmEvau5XDEAlvFHzjmueXClCSeY%2BcXyuDnxpGGUYAGSwCiPZfk8mSnuBudAANqx%2FQWpRN7NIZWml9OtVzbS8eQ5V14Xla6ykr%2BZEhyzqOYsH%2F4L7S178ty0FY0KW6hBUyQgCSbSDngWSI%2FdaqJ%2FXadIbKlUcjR7JPDM%2Bds%2F8Gw9c79ckW8k72NZ7Zm%2B%2BNeONIhV%2FsXOtBms4cDuZ0XMNq%2F8KgGOpwB6SMup4p3FPGT1Oeanbig542TIVVZVoIDHl5Z72F%2FebMQupiA55NV6hzaivhXzNUnbGq0Ci9ghwDqCjnPSVUmNAWRVf1EYpDFd5zHUDfhCw43J%2BIZeUkInUBAxVA4RmvEqsoVUvA%2FZ21BQlYc3oBqFZ2%2F9oXI9JETONCncmJtUxF0n82bvXjKRdPtbf45PxJj4NK3THZGDSjf6DLNOr download the ZIP archive:DownloadClose**